examples/Nat/NAT.cs

```diff
@@ -42,17 +42,39 @@
 
 namespace Pax.Examples.Nat
 {
+  /// <summary>
+  /// A packet processor that performs Network Address Translation (a NAT).
+  /// </summary>
   public sealed class NAT : SimplePacketProcessor
   {
+    /// <summary>
+    /// A value indicating the packet should be dropped.
+    /// </summary>
     public const int Port_Drop = -1;
+    /// <summary>
+    /// A value representing the network interface that faces outside.
+    /// </summary>
     public const int Port_Outside = 0;
 
     private readonly IPAddress my_address;
     private readonly PhysicalAddress next_outside_hop_mac;
     private readonly TimeSpan connection_timeout;
     private readonly Timer gcTimer;
-
-    public NAT (IPAddress my_address, PhysicalAddress next_outside_hop_mac, ushort tcp_start_port, ushort udp_start_port, TimeSpan? connection_timeout = null) {
+    
+    // Use a separate namespace for each transport protocol
+    private SpecialisedNAT<TcpPacket> tcpNat;
+    private SpecialisedNAT<UdpPacket> udpNat;
+
+    /// <summary>
+    /// 
```

Comment:
This `summary` block seems to be redundant, unless it was intended to host a comment.